ddtrace/__init__.py

```diff
@@ -1,13 +1,17 @@
 from ._monkey import patch  # noqa: E402
 from ._monkey import patch_all
 from .internal.utils.deprecation import deprecated  # noqa: E402
+from .logger import configure_logger
 from .pin import Pin  # noqa: E402
 from .settings import _config as config  # noqa: E402
 from .span import Span  # noqa: E402
 from .tracer import Tracer  # noqa: E402
 from .version import get_version
 
 
+configure_logger()
```

Comment:
we should hoist this up like we suggest for `import ddtrace; ddtrace.patch_all()`.

it will be important to ensure logging is setup before we could log our first log line.

Reply:
I tried doing something like this in the latest commit, but one of the precommit hooks didn't like that, let me know if there's a better way to format this!

Right now I have something like this:
```
from .logger import configure_ddtrace_logger  # noqa: E402


configure_ddtrace_logger()  # noqa: E402
``